### ISSUE TITLE
Facility Locators Dashboard: V2 Enhancements

### ISSUE THREAD

**Issue by johnny-jesensky-adhoc** [department-of-veterans-affairs/va.gov-team#76514]

**Comment by johnny-jesensky-adhoc** [comment]
Reach out to Michelle letting them know that FE can enable the _no results found_ tracking.

**Comment by johnny-jesensky-adhoc** [comment]
Per refinement, JJ to break this out into three separate tasks -- look to refine separately.

**Comment by jwoodman5** [comment]
@Blivaditis @ChristineYoungZA Is this still relevant? I would presume no, but would like some validation. Thanks.

**Comment by Blivaditis** [comment]
I don't believe so, I haven't heard mention of this dashboards since last March.

**Comment by AdamOye** [comment]
@Blivaditis @ChristineYoungZA  Can I delete this ticket per comments above?

**Comment by AdamOye** [comment]
This is a follow up ticket to #61633

**Comment by AdamOye** [comment]
Cont. ticket to Facility request #61633 This can be closed. due to confirmation form client
